src/validator/userInputMoneyValidator.ts

```diff
@@ -0,0 +1,21 @@
+import { ERROR_MESSAGE, CONFIGURATION } from '../constants';
+
+const userInputMoneyValidator = {
+  isOverMax(currentInputMoney: number, userInputMoney: number) {
+    return currentInputMoney + userInputMoney > CONFIGURATION.INPUT.MAX;
+  },
+
+  isIncorrectUnit(inputMoney: number) {
+    return inputMoney % CONFIGURATION.AMOUNT.UNIT !== 0;
+  },
+};
+
+export const validateInputMoney = (money: number, userInputMoneyAmount: number): void => {
```

Comment:
```suggestion
export const validateInputMoney = (money: number, userInputMoneyAmount: number) => {
```
return을 따로 코드라인에 작성하지 않아서 void타입을 자동으로 추론합니다 🙂 
(나머지 validator 로직도 마찬가지에요)

Reply:
반영 링크입니다! [829ee34](https://github.com/woowacourse/javascript-vendingmachine/pull/56/commits/829ee344918bb5da224cf0417f89441c850f99e9)